app/v2/controllers/V2DeparturesController.scala

```diff
@@ -64,13 +63,15 @@ class V2DeparturesControllerImpl @Inject() (
   temporaryFileCreator: TemporaryFileCreator,
   authActionNewEnrolmentOnly: AuthNewEnrolmentOnlyAction,
   validationService: ValidationService,
-  messageSizeAction: MessageSizeAction
+  departuresPersistenceService: DeparturesPersistenceService,
```

Comment:
Personally I think the addition of Persistence is a little redundant, IMO either a service that is responsible for Persistence (in which case, it could just be a connector) or a Departures Service.